demos/tests/run_tests.py

```diff
@@ -333,7 +333,10 @@ def option_to_args(key, value):
                             execution_time = timeit.default_timer() - start_time
                             demo.parse_output(output, test_case, device)
                         except (subprocess.CalledProcessError, subprocess.TimeoutExpired) as e:
-                            output = e.output
+                            if type(e) == subprocess.TimeoutExpired:
```

Comment:
I think it's handled by `elif isinstance(e, subprocess.TimeoutExpired):` below

Reply:
Yes. The exception found serval days ago can't be reproduced now. So Close this PR now.

Reply:
Failure appeared again as below caused by the timeout. I'm not sure if reopen this PR again? @Wovchena
![image](https://user-images.githubusercontent.com/59246673/189798611-2392c6b8-1320-4bb5-a237-0903c252f040.png)